cmake/Modules/FindBLIS.cmake

```diff
@@ -0,0 +1,75 @@
+# - Find BLIS library
+#
+# This module sets the following variables:
+#  BLIS_FOUND - set to true if a library implementing CBLAS interface is found.
+#  BLIS_INCLUDE_DIR - path to include dir.
+#  BLIS_LIB - list of libraries for BLIS.
+#
+# CPU only Dockerfile to build with AMD BLIS is available at the location
+# pytorch/docker/pytorch/cpu-blis/Dockerfile
+#
+# Example commandline to build PyTorch with AMD BLIS:
+# sudo DOCKER_BUILDKIT=1 docker build . -t docker-image-repo-name
+# Example commandline to run the built docker container:
+# sudo docker run --name container-name -it docker-image-repo-name
+
+
+SET(BLIS_INCLUDE_SEARCH_PATHS
+  /usr/include/blis
+  /usr/local/include
+  /usr/local/include/blis
+  /opt/blis/include
+  $ENV{BLIS_HOME}
+  $ENV{BLIS_HOME}/include
+)
+
+SET(BLIS_LIB_SEARCH_PATHS
+  /lib/blis
+  /lib64/blis
+  /usr/lib/blis
+  /usr/lib64/blis
+  /usr/local/blis/lib
+  /opt/blis/lib
+  $ENV{BLIS}
+  $ENV{BLIS}/lib
```

Comment:
INCLUDE and LIB search paths should be controlled by the same environment variables. include is only controlled by BLIS_HOME, so BLIS is not needed in library search path, is it?
```suggestion
```